Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Transformer.purs

```diff
@@ -545,7 +545,7 @@ filterWithFareAndVariant state estimates estimateAndQuoteConfig preferredVariant
         map (\sortedEstimate -> sortedEstimate.item) sortedEstimates
 
   filterEstimateByVariants :: Array String -> Array EstimateAPIEntity -> Array EstimateAPIEntity
-  filterEstimateByVariants variant estimates = DA.take 1 (sortEstimateWithVariantOrder (DA.filter (\(EstimateAPIEntity item) -> DA.any (_ == item.vehicleVariant) variant) estimates) variant)
+  filterEstimateByVariants variant estimates = DA.take 1 (sortEstimateWithVariantOrder (DA.filter (\(EstimateAPIEntity item) -> DA.any (_ == fromMaybe "" item.serviceTierName) variant) estimates) variant)
```

Comment:
we are matching variant type with serviceTierName therefore it is filtering every values